src/components/bookmark-button/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { RemoveBookmarkIcon, AddBookmarkIcon } from './icons'
+import s from './bookmark-button.module.css'
+import useAuthentication from 'hooks/use-authentication'
+
+/**
+ * TODO
+ * - create a 'plugged' in HOC that can render modals, update bookmark state etc.
+ * This HOC will be used by this button and the one in tutorial meta
+ */
+
+interface BookmarkButtonProps {
+	isBookmarked: boolean
+}
+
+export default function BookmarkButton({ isBookmarked }: BookmarkButtonProps) {
+	const { isAuthEnabled } = useAuthentication()
+	// NOTE! - hiding this component from prod until auth is enabled
+	if (!isAuthEnabled) {
+		return null
+	}
```

Comment:
Sorry for any confusion on this via Slack! I meant to suggest using the `AUTH_ENABLED` constant from `use-authentication`, which can be imported directly:

```
import { AUTH_ENABLED } from 'hooks/use-authentication'
```

Otherwise, invoking the `useAuthentication` hook will cause some unnecessary JavaScript to run here.